disco_aws_automation/disco_rds.py

```diff
@@ -461,3 +467,50 @@ def cleanup_snapshots(self, days):
                 snapshot_id = snapshot['DBSnapshotIdentifier']
                 logging.info("Deleting Snapshot %s since its older than %d", snapshot_id, days)
                 self.client.delete_db_snapshot(DBSnapshotIdentifier=snapshot_id)
+
+    def _get_instance_identifier(self, env_name, database_name):
+        return env_name + '-' + database_name
+
+    def _get_database_name(self, env_name, instance_identifier):
+        identifier_prefix = len(env_name + '-')
+        return instance_identifier[identifier_prefix:]
+
+    def _get_db_instance(self, instance_identifier):
+        try:
+            return self.client.describe_db_instances(DBInstanceIdentifier=instance_identifier)
+        except botocore.exceptions.ClientError:
+            return None
+
+    def clone(self, env_name, database_name):
```

Comment:
nit: `env_name` -> `source_vpc` and `database_name` -> `source_db`. I feel this makes it more clear.